docs/concepts/add-in-development-best-practices.md

```diff
@@ -20,7 +20,10 @@ Effective add-ins offer unique and compelling functionality that extends Office
   - Embed complementary services within Office applications.
   - Improve the Office experience to enhance productivity.
 - Make sure that the value of your add-in is clear to users right away by [creating an engaging first-run experience](#create-an-engaging-first-run-experience).
-- Create an [effective AppSource listing](/partner-center/marketplace-offers/create-effective-office-store-listings). Make the benefits of your add-in clear in your title and description. Don't rely on your brand to communicate what your add-in does.
+- Create an [effective AppSource listing](/partner-center/marketplace-offers/create-effective-office-store-listings).
```

Comment:
It's curious to me that "create an AppSource listing" is in the "planning" phase. I don't think it's wrong to approach developing an add-in by starting with the value prop, but this framing makes it seem like you should write your store listing before the code.

Reply:
@AlexJerabek Thoughts?